package.json

```diff
@@ -89,7 +89,7 @@
     "tar": "^4.4.4",
     "traverse": "^0.6.6",
     "uuid": "^3.1.0",
-    "yargs": "^8.0.2"
+    "yargs": "^12.0.5"
```

Comment:
Last time I tried updating yargs, a number of things stopped working, though there have been a couple major updates since then. Have you seen any problems with it?

Reply:
For now i've tried a lot (but not all) of the command launchable by mup, no error so far.
I'll try a lot for with the changes requested builder.hide(commandName)